lib/script_base.rb

```diff
@@ -54,6 +54,7 @@ def run
 
     stderr.puts "*Task*: `#{result.subtask}`"
     stderr.puts "*UUIDs*: #{result.uuids.map { |uuid| "`#{uuid}`" }.join(', ')}"
+    stderr.puts "*Messages*: `#{result.table.filter_map { |row| "#{row[1]} : #{row[0]}" if row[0]!='uuid'}.join('\n') }`"
```

Comment:
1. let's only add `*Messages*` if there are messages, and let's make that a separate property of `result`
2. let's use array deconstruction  to make this a little clearer
3. I would put the UUIDs first?
4. Can we try out the list formatting, ex:

```markdown
*Messages*:
* `uuid1`: Cannot do something
* `uuid2`: Cannot do something else
```

```suggestion
    if result.messages.present?
      stderr.puts "*Messages*:
      stederr.puts result.messages.map { |uuid, message| "* `#{uuid}`: #{message}" }.join("\n")
    end
```

Reply:
I got rid of the separate property because everything is already in `table` - looking through the code, I thought there was always a table row (with accompanying messages), but you might be right, so I’ll put a conditional in there.

Reply:
I think it's clearer to have a separate property for logging like this, because then we dont have to make any assumptions about the structure of the table, or if we ever put UUID in a different column in the future

Reply:
I have added a separate property